src/Symfony/Component/Mime/Tests/Encoder/Rfc2231EncoderTest.php

```diff
@@ -102,15 +102,15 @@ public function testEncodingAndDecodingSamples()
         $dir = realpath(__DIR__.'/../Fixtures/samples/charsets');
         $sampleFp = opendir($dir);
         while (false !== $encoding = readdir($sampleFp)) {
-            if ('.' == substr($encoding, 0, 1)) {
```

Comment:
I wonder why the equality test ignored the type of the variables (no `===`).

Reply:
There is actually no difference between triple `===` and double `==` when both operands has the same type.